app/models/host/managed.rb

```diff
@@ -239,6 +239,20 @@ def built(installed = true)
     end
   end
 
+  # rebuilds TFTP, DHCP, DNS for the host
+  def rebuild_config
+    result = {:tftp => false, :dhcp => false, :dns => false}
```

Comment:
Any chance we can move this into the orchestration concerns somehow?  A bit like how they queue methods already, we could have a generic way to rebuild orchestration entries.